### DISCUSSION TITLE
Are existing renewable plants taken into account ?

### DISCUSSION THREAD

**Discussion by dhoop-iidev** [pypsa-meets-earth/pypsa-earth#915] [General]
Hello everybody

When I downloaded earlier versions of pypsa earth, i know that only existing conventional power plants were retrieved from various open source datasets using powerplantmatching (ppm). But I realized that this tool is now capable to include renewable plants such as wind and solar. But they are still not included right ?

I understand that having none of the existing infrastructure might be interesting to understand if the current investments in renewable have been done wisely, but that could be just an option to toggle in the config, no ?

If you could provide some light on this topic, I would be very grateful !

Gaspard

**Comment by davide-f** [comment]
Hello @dhoop-iidev !
We have been discussing very recently and today as well about that.
It is an interesting topic to investigate and indeed we want to support that :)
Would you like to investigate that and verify to include those powerplants? :)

See https://github.com/pypsa-meets-earth/pypsa-earth/issues/753

**Comment by dhoop-iidev** [comment]
That's nice to hear :) I'll check if I can contribute in any way
